docs/kqueen.rst

```diff
@@ -52,7 +52,9 @@ Serializers
 Configuration
 --------------
 
-Example configuration files are located in `config/` directory. Default configuration file is `config/dev.py` and this can be configured by `KQUEEN_CONFIG_FILE` environment variable. Any environment variable matching name `KQUEEN_*` will be loaded and merged with configuration file.
+Example configuration files are located in the `config/` directory. Default configuration file is `config/dev.py`.
```

Comment:
Examples of *** ?

Reply:
Examples of configuration files/Configuration file examples ... The default configuration file is `config/dev.py`.